core/src/main/java/com/sopt/core/designsystem/component/calendar/NoostakCalendar.kt

```diff
@@ -0,0 +1,277 @@
+                modifier = Modifier
+                    .size(16.dp)
+                    .clickable {
+                        if (month == 12) {
+                            year += 1
+                            month = 1
+                        } else {
+                            month += 1
+                        }
+                    }
+            )
+        }
+
+        Spacer(modifier = Modifier.height(30.dp))
+
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            listOf("일", "월", "화", "수", "목", "금", "토").forEach { day ->
+                Text(
+                    text = day,
+                    style = typography.c2SemiBold,
+                    textAlign = TextAlign.Center,
+                    color = colors.gray600,
+                    modifier = Modifier.weight(1f)
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.height(16.dp))
+        Column(verticalArrangement = Arrangement.spacedBy(15.dp)) {
+            var day = 1
+            for (week in 0..5) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    (0..6).forEach { dayOfWeek ->
+                        val dateText = when {
+                            week == 0 && dayOfWeek < firstDay -> ""
+                            day > totalDays -> ""
+                            else -> (day++).toString()
+                        }
+
+                        val dateValue = "$year-${month.toString().padStart(2, '0')}-${dateText.padStart(2, '0')}"
+
+                        val isSelected = dateValue in selectedDates
+                        val isRange = !isSingleDate &&
+                                dateText.isNotEmpty() &&
+                                startDate.isNotEmpty() &&
+                                endDate.isNotEmpty() &&
+                                LocalDate.parse(dateValue) in LocalDate.parse(startDate)..LocalDate.parse(endDate)
+
+                        val isStart = dateValue == startDate
+                        val isEnd = dateValue == endDate
+
+                        Box(
+                            modifier = Modifier
+                                .weight(1f)
+                                .height(40.dp),
+                            contentAlignment = Alignment.Center
+                        ) {
+                            when {
+                                isSelected -> {
+                                    Box(
+                                        modifier = Modifier
+                                            .size(40.dp)
+                                            .background(
+                                                color = colors.blue300,
+                                                shape = androidx.compose.foundation.shape.CircleShape
```

Comment:
P3 : 이렇게 간단히 할 수도 있슴다
```suggestion
                                                shape = CircleShape
```